scripts/check_doc_links.py

```diff
@@ -141,7 +145,7 @@ def main() -> None:  # pylint: disable=too-many-locals
         if broken_links:
             broken_links_str = "\n".join(
                 [
-                    f"{file_name}: {[url[1] for url in urls]}"
+                    f"{file_name}: {[(url[1], url[2]) for url in urls]}"
```

Comment:
Show the status code next time it fails.

Reply:
Future devs and self would need to study the `broken_links` in order to understand what these indexes are accessing. A `dataclass` or a `dict` would have been more helpful. However, we could at least add a comment on the code with what these indexes are accessing.

Reply:
Fixed